src/Components/CreateImageWizard/validators.ts

```diff
@@ -33,13 +33,23 @@ export const isGcpEmailValid = (gcpShareWithAccount: string | undefined) => {
 };
 
 export const isMountpointMinSizeValid = (minSize: string) => {
+  if (!minSize) {
+    return false;
+  }
   return /^\d+$/.test(minSize) && parseInt(minSize) > 0;
 };
 
-export const isBlueprintNameValid = (blueprintName: string) =>
-  blueprintName.length >= 2 &&
-  blueprintName.length <= 100 &&
-  /\w+/.test(blueprintName);
+export const isBlueprintNameValid = (blueprintName: string) => {
+  if (!blueprintName) {
+    return false;
+  }
```

Comment:
This might be better in `useDetailsValidation`, because then we could add a custom error message that could be something like: 'Blueprint name is required'